torch/_dynamo/graph_deduplication.py

```diff
@@ -154,6 +155,9 @@ def _replace_region_with_subgraph(
         for node in reversed(region):
             graph.erase_node(node)
 
+    if config.graph_deduplication_lint:
```

Comment:
This seems like it should be on by default. Maybe you need this for a time being but we should remove the config at the end.

Reply:
Yeah I can turn it on by default in an upcoming stack.